README.rst

```diff
@@ -1,83 +1,74 @@
-Documentation Build Instructions
+Documentation build instructions
 ################################
 
-The `website documentation`_ for Clear Linux\* OS for Intel Architecture
-should be written in :abbr:`ReStructuredText (ReST)` AKA ``.rst``, which
-makes it easy to build parsable, command-line readable, indexed, and
-search-friendly documentation and APIs with `Sphinx`_.
-
-.. _requirements:
-
-Requirements
-============
-
-To build documentation with Sphinx, ensure your system has these
-prerequisites:
-
-* `GNU make`_
-* `Python`_
-* `PIP`_
-* `Sphinx`_
-
-The instructions for installing these varies according to OS. On a basic out-
-of-the-box Ubuntu-like OS (which usually has Python installed by default),
-check your python version you might need something like:
-
-.. code-block:: console
-
-   $ sudo apt-get install python-pip
-   $ sudo pip install -U sphinx sphinx-autobuild
+.. todo add comment re not using standards here.
 
+`Clear Linux\* OS documentation`_ is written using `reStructuredText`_ and
+built using `Sphinx`_. Documentation may be built locally for development and
+testing purposes by following the instructions in this README.
 
```

Comment:
Suggest reordering sentence:  Follow the instructions in this README to build the documentation locally for development and testing.

Reply:
Done.